### ISSUE TITLE
ListPicker Selection change not working properly when we set the selectedList using setState

### ISSUE THREAD

**Issue by jyothsnag1987** [SharePoint/sp-dev-fx-controls-react#315]
#### Category
[ ] Enhancement

[X] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [   1.13.2   ]

#### Expected / Desired Behavior / Question
_On selection change, selectedList should be highlighted and options should not be repeated_

#### Observed Behavior
_On ListPicker selection change, selectedList is getting highlighted but the options are binding again_
![Capture](https://user-images.githubusercontent.com/49554189/58427848-14e2ab00-80a1-11e9-869a-86db9d8c2f8b.PNG)

#### Steps to Reproduce
_After migrating my project from SPFx 1.6 to SPFx 1.8.2 I started seeing this issue.
If we don't set the selectionList from state then we wont have issue. But in my scenario I need to set the selectedList from state and it should be stored back to the state on selectionChanged event_

**Comment by justdevelopment** [comment]
After some research we think it's related to the control [pushing all found lists to the _options](https://github.com/SharePoint/sp-dev-fx-controls-react/blob/master/src/controls/listPicker/ListPicker.tsx#L79), without checking if the item is already in this list. As this function is also called on any change done to the selected item it will duplicate all items again and again on change. It's unclear to us why this issue started after upgrading our project.

A proposed solution would be to check if the item is already in the list before pushing;
```ts
results.value.map(list => {
        if (!this.optionsContainsList(list)) {
          this._options.push({
            key: list.Id,
            text: list.Title
          });
        }
      });

**Comment by estruyf** [comment]
This should have been fixed. You can test it out in the latest beta version and will be available in `1.14.0`. More information on how you can test out a beta version can be found here: https://sharepoint.github.io/sp-dev-fx-controls-react/beta/

**Comment by jyothsnag1987** [comment]
Thank you for providing the fix, Its working fine with beta version. Could we know the planned target date for 1.14.0 release.

**Comment by estruyf** [comment]
Release date would be later this week or next week, depends on the other fixes that are waiting to be verified.

**Comment by estruyf** [comment]
This is now included within the `1.14.0` release
